.env.example

```diff
@@ -12,3 +12,7 @@ PORT=4000
 COVERALLS_REPO_TOKEN=covernal_token
 # PRODUCTION DB: REMOTE
 DATABASE_URL_PROD=production_url
+
+# SECRET TO BE USED IN THE HASHING PASSWORD AND IN TOKENS
+ACCESS_TOKEN_SECRET=766eb85b713df63f7285459288b790c8e9e6ab43229de9dca33fe08c1c5cc30961a83456c59c9aeefafc1db3bb8d27f6964572a456fec35d2d353874409ccc78
+REFRESH_TOKEN_SECRET=a00a248ce550b46b66b08cf5ed597a0b14b1877c02714110bfe402ded54a4a3e76b426c8da9f65726c33dbe83f736ed9d905bafd4a7e942bb0e271bd499a7c6b
```

Comment:
I hope these tokens are not valid ones, otherwise, we exposed them to the public.